docs/src/intro/usage.md

```diff
@@ -65,23 +65,60 @@ This will dump the output of the program into the `fib.out` file. The output fil
 
 ### Miden Debugger
 
-The Miden debugger is a shell that allow for efficient debugging of miden assembly programs that are sourced from file.  A Miden assembly program file and inputs are specified when the debugger is instantiated.  The debugger allows the user to step through the execution of the program with clock cycle granularity and provides the ability for virtual machine state inspection at each clock cycle.  The Miden debugger supports the following commands:
-
-```
-!next        steps to the next clock cycle
-!play        executes program until completion or failure
-!play.n      executes n clock cycles
-!prev        steps to the previous clock cycle
-!rewind      rewinds program until beginning
-!rewind.n    rewinds n clock cycles
-!print       displays the complete state of the virtual machine
-!stack       displays the complete state of the stack
-!stack[i]    displays the stack element at index i
-!mem         displays the complete state of memory
-!mem[i]      displays memory at address i
-!clock       displays the current clock cycle
-!quit        quits the debugger
-!help        displays this message
+The Miden debugger is a command-line interface (CLI) application, inspired on [GNU gdb](https://sourceware.org/gdb/), that allows debugging of Miden assembly (MASM) programs. The debugger allows the user to step through the execution of the program, both forward and backward, either per clock cycle tick, or via breakpoints.
+
+The Miden debugger supports the following commands:
+
+| Command | Shortcut | Arguments | Description |
+| --- | --- | --- | --- |
+| next | n | <count> | Steps `count` clock cycles. Will step `1` cycle of `count` is ommitted. |
+| continue | c | - | Executes the program until completion, failure or a breakpoint. |
+| back | b | <count> | Backward step `count` clock cycles. Will back-step `1` cycle of `count` is ommitted. |
+| rewind | r | - | Executes the program backwards until the beginning, failure or a breakpoint. |
+| print | p | - | Displays the complete state of the virtual machine. |
+| print mem | p m | <address> | Displays the memory value at `address`. If `address` is ommitted, didisplays all the memory values. |
+| print stack | p s | <index> | Displays the stack value at `index`. If `index` is ommitted, displays all the stack values. |
+| clock | c | - | Displays the current clock cycle. |
+| quit | q | - | Quits the debugger. |
+| help | h | - | Displays the help message. |
```

Comment:
nit: things like `<count>` are not displayed (at least in Github rendering). Maybe if we put backticks around them it'll work?

Reply:
Fixed!